### PR TITLE
openstackclient: Get rid of kolla_start

### PR DESCRIPTION
**Pull request by kajinamit**
... so that we don't need to allow the cloud-admin users to use sudo. Usage of kolla_start is redundant because we have no config files copied into the container by the command.

### CODE REVIEW

File: pkg/openstackclient/funcs.go [hunk 1]
```diff
@@ -54,8 +54,10 @@ func ClientPod(
 	clientPod.Spec.ServiceAccountName = ServiceAccount
 	clientPod.Spec.Containers = []corev1.Container{
 		{
-			Name:  "openstackclient",
-			Image: instance.Spec.ContainerImage,
+			Name:    "openstackclient",
+			Image:   instance.Spec.ContainerImage,
+			Command: []string{"/bin/sleep"},
+			Args:    []string{"infinity"},
 			SecurityContext: &corev1.SecurityContext{
```
Comment (kajinamit) [inline pkg/openstackclient/funcs.go:61]: We can also remove these and anyuid, but I'd leave it for follow-up.

**Comment by gibizer** [comment]
Looks OK to me but we have no e2e deploy job for this repo yet. Have you deployed this change?

**Review by gibizer — APPROVED** [review]
I successfully run this alone as well as together with the tcib change in https://github.com/openstack-k8s-operators/openstack-operator/pull/232

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**